### ISSUE TITLE
Impossible de récupérer les enfants d'une zone par API

### ISSUE THREAD

**Issue by antoine-de** [etalab/data.gouv.fr#163]
## Description du bug

L'[API de récupération](https://doc.data.gouv.fr/api/reference/#/spatial/spatial_zone_children) des enfants d'une zone `/spatial/zone/:id/children` n'a pas l'air de fonctionner.

## Comment reproduire le bug
les urls suivantes renvoient une erreur 500 :

https://www.data.gouv.fr/api/1/spatial/zones/fr:epci:241700459@1999-01-01/children
ou bien
https://www.data.gouv.fr/api/1/spatial/zones/country:fr/children

## Contournement
Pour contourner le problème il est possible d'utiliser le fichier [geozones](https://www.data.gouv.fr/fr/datasets/geozones/#_) pour reconstruire la hierarchie administrative.

Dans mon cas j'avais uniquement besoin de la hierarchie des [EPCI](https://fr.wikipedia.org/wiki/%C3%89tablissement_public_de_coop%C3%A9ration_intercommunale), j'ai pu utiliser [un extrait d'etalab du découpage pour les EPCI](https://unpkg.com/@etalab/decoupage-administratif/data/epci.json).

**Comment by agarrone** [comment]
@abulte cassé depuis l'extinction géo ?

**Comment by abulte** [comment]
Non, pas de rapport. C'est la maintenance des geozones qui est en cause.

**Comment by agarrone** [comment]
Pas d'investissement prévu sur ce sujet pour l'instant. 
Refacto de la gestion des zones à venir.
